### ISSUE TITLE
zip64 support

### ISSUE THREAD

**Issue by quininer** [quininer/unzipx#1]

**Comment by paulie-g** [comment]
Ran into this trying out unzrip. Most large zip files today -- the reason someone would want to use a parallel-capable unarchiver -- are zip64.

**Comment by quininer** [comment]
It's not too hard to implement, but I'm mostly busy with totk and other projects now.

If anyone needs it, I'd welcome a PR.

**Comment by paulie-g** [comment]
Fair enough. I was just leaving a +1 since the issue was open. 7zip has a parallel implementation that worked well enough for me.

**Comment by ghuls** [comment]
The `zip64` branch supports it now to some extent.

Not sure how if memory usage `unzipr` is supposed to match the size of the zip file or that it only happens if you have more RAM than the file is big:

```
Maximum resident set size (RSS: memory) (kiB) of unzipr: 61_335_367_680 bytes
zip64 filesize: 61_327_644_470 bytes (58 GB)
```

Memory usage grows steadily during extracting of the zip file (with files with sizes between 125MB - 4.8G) till it reaches the filesize of the zip file when it extracts the last file from the archive.

**Comment by quininer** [comment]
@ghuls This is normal, unzrip uses mmap to load files into memory. This is only virtual memory, not actual memory usage.
